library/src/morpheus-master/sass/modules/_toolmenu.scss

```diff
@@ -491,15 +489,12 @@ nav#subSites{
 			&.is-current{
 				position: relative;
 				margin-right: -1px;
+				border-left: $tool-menu-item-selected-border-left;
```

Comment:
I don't like this implementation because it will make the text of the tool name shift to the right to make room for the border. Which that kind emphasizes the tab effect, it will cause jitter for all tools and unexpected consequences for long tool names that just fit the width when they are not selected (because the shift may cause them to span two lines when they are the selected tool).